### ISSUE TITLE
Makes the same sound on macOS regardless of parameters

### ISSUE THREAD

**Issue by wmcbrine** [vareille/toot#5]
Same duration as well. macOS 11.6, MacBook Air M1 (although I think I had the same result on an x86 Pro).

**Comment by vareille** [comment]
yes, on macs, you need to install "sox" from homebrew for this to work as intended

https://formulae.brew.sh/formula/sox#default
